feature/client-api/src/test/java/com/simprints/feature/clientapi/mappers/request/IntentToActionMapperTest.kt

```diff
@@ -103,6 +103,16 @@ class IntentToActionMapperTest {
         }
     }
 
+    @Test
+    fun `correctly handles CommCare intent with blank session ID`() = runTest {
```

Comment:
Maybe add another test for missing sessionId as it's the real world scenario currently.

Reply:
It was added in the initial fix. One test case above this one.